app/src/main/java/com/invisibleteam/goinvisible/mvvm/edition/EditActivity.java

```diff
@@ -85,10 +87,14 @@ public static Intent buildIntent(Context context, ImageDetails imageDetails) {
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     final EditTagListener editTagListener = new EditTagListener() {
         @Override
-        public void openEditDialog(Tag tag) {
+        public void openTagEditionView(Tag tag) {
             if (tag.getKey().equals(ExifInterface.TAG_GPS_LATITUDE)) {
                 EditActivity.this.tag = tag;
                 startPlaceIntent();
+            } else if (tag.getTagType().getInputType().equals(InputType.UNMODIFIABLE)) {
+                showSnackBar(R.string.unmodifiable_tag_message);
+            } else if (tag.getTagType().getInputType().equals(InputType.INDEFINITE)) {
```

Comment:
better approach is comparing enum value with sth - we are securing against NUllPointerException